gae/js/app/views/sitecaptain.js

```diff
@@ -68,11 +68,15 @@ define(
                 m.destroy();
                 this.render();
             },
-            addCaptain: function() {
-                console.log('add captain ', this);
+            addCaptain: function(e) {
+                e.preventDefault();
                 var self = this;
+                var choices = this.form.fields.models.find(function(model){return model.get('name') == 'captain';});
+                var captain_id = this.model.get('captain');
+                var captain  = _.findWhere(choices.get('options'), {value: captain_id});
```

Comment:
this is where you need to grab the captain label.  then it's a string and you have a safe copy.  the reference to captain is an object and could change before the callback is invoked. 

var captain_label = ..... .label;

Reply:
I GET it !!!! You are so patient Luke!!